openmdao/solvers/nonlinear/newton.py

```diff
@@ -190,6 +192,25 @@ def _iter_execute(self):
         # Disable local fd
         approx_status = system._owns_approx_jac
         system._owns_approx_jac = False
+        
+        # This cycles through the solvers of the subsystems once at the start of the problem
+        if self.options['initial_sub_solve'] == True:
```

Comment:
I think it's unintuitive behavior for the code to set initial_sub_solve to False after running it once. That means that when calling run_model multiple times, initial_sub_solve would only be used the first time, which is not what I would expect as a user. This should either be explained in the options documentation, or we find an alternate implementation.
That would be to override `__call__`, put the `initial_sub_solve` code in there, then call super's `__call__` or _run_iterator at the end. This way multiple consecutive calls of run_model would all run the initial_sub_solve